website/src/lib/plugin-get-static-props.ts

```diff
@@ -47,7 +47,5 @@ export const pluginGetStaticProps = (fileName: string) => async () => {
         compiledHeader: mdxHeader.result,
       },
     },
-    // The page will be considered as stale and regenerated every 24 hours.
-    revalidate: 60 * 60 * 24,
```

Comment:
do we not want to re-generate?

Reply:
It never worked because we use `next export` 😅